features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -296,17 +321,26 @@ function getSubscriberCountByAPIs(providerName, isAllStatistics) {
 function getProviderAPIVersionUserUsage(providerName,apiName,version,serverUrl) {
     var usage = [];
     var usageout = [];
+
+    if(usageClient == null){
+        return {
+            error:false,
+            usage:[]
+        };
+    }
+
     try {
-        usage = apiProvider.getProviderAPIVersionUserUsage(providerName,apiName,version,serverUrl);
+        usage = usageClient.getUsageBySubscribers(providerName,apiName,version,10);
         if (log.isDebugEnabled()) {
             log.debug("getProviderAPIVersionUserUsage for : " + providerName);
         }
         if (usage == null) {
             return {
-                error:true
+                error:false,
+                usage:[]
             };
-
         } else {
+            usage=JSON.parse(gson.toJson(usage));
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea https://github.com/wso2/carbon-apimgt/commit/d7e56b1d85254369d07aab9572f13d4c2120df97